### ISSUE TITLE
Clone from templates

### ISSUE THREAD

**Issue by mkuzmin** [LizaTretyakova/packer-builder-vsphere#6]
if a source VM is a template
<img width="152" alt="screen shot 2017-05-08 at 18 18 39" src="https://cloud.githubusercontent.com/assets/712402/25811108/d46e33b6-341a-11e7-9e53-d358540aa8dd.png">

then builds fail with an error:
```
==> vsphere: start cloning...
Build 'vsphere' errored: A specified parameter was not correct. 
spec.location.pool
```

**Comment by mkuzmin** [comment]
related to #4

**Comment by LizaTretyakova** [comment]
Fixed in #14.
